From b408dd404f291af137c5751dd776252ad5a09651 Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Wed, 12 Aug 2026 22:12:00 +0200
Subject: [PATCH 08/17] base.pro: Remove building of tests

Solves the issue with incorrect LS2 ACG files in the manifest file for luna-surfacemanager.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>

Upstream-Status: Pending
---
 base/base.pro | 1 -
 1 file changed, 1 deletion(-)

diff --git a/base/base.pro b/base/base.pro
index 17efbd2..ffc2de6 100644
--- a/base/base.pro
+++ b/base/base.pro
@@ -20,7 +20,6 @@ SUBDIRS = \
     src \
     qml \
     startup \
-    tests \
     utils
 
 cursor_theme {
